trade-finance-manager-ui/server/routes/login/configs/loginNonSso.ts

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable @typescript-eslint/no-misused-promises */
```

Comment:
Can we remove this?

Reply:
No -- is present in all TS files for routes